### PR TITLE
docs: fix code formatting and add link to es5 serve section

### PR DESCRIPTION
**Pull request by alan-agius4**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by kapunahelewong — APPROVED** [review]
Nice work! Thank you ✨

**Comment by alan-agius4** [comment]
Caretaker: kindly note window ci failure is unrelated

**Review by petebacondarwin — APPROVED** [review]
I think the apache config snippet needs fixing. Otherwise LGTM.
In any case it is better than what we have now.

In general my recommendation is for small snippets to be simply wrapped in triple backticks and for longer snippets to be pulled out into external files.

File: aio/content/guide/deployment.md [hunk 1]
```diff
@@ -161,7 +160,6 @@ The list is by no means exhaustive, but should provide you with a good starting
 
     &#35 If the requested resource doesn't exist, use index.html
     RewriteRule ^ /index.html
-
   </code-example>
```
[the remark this answers is not in the document]
Comment (alan-agius4) [reply aio/content/guide/deployment.md]: Any ideas how to remove the spacing?

File: aio/content/guide/deployment.md [hunk 2]
```diff
@@ -152,36 +152,31 @@ The list is by no means exhaustive, but should provide you with a good starting
...
-    &#35 If an existing asset or directory is requested go to it as it is
-    RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} -f [OR]
-    RewriteCond %{DOCUMENT_ROOT}%{REQUEST_URI} -d
-    RewriteRule ^ - [L]
-
-    &#35 If the requested resource doesn't exist, use index.html
-    RewriteRule ^ /index.html
-
-  </code-example>
+  &#35 If the requested resource doesn't exist, use index.html
```
Comment (petebacondarwin) [inline aio/content/guide/deployment.md]: Try deleting the blank line above this

Comment (petebacondarwin) [reply aio/content/guide/deployment.md]: If you want there to be a blank line in the rendered output then add some `<br>` tags.

Comment (alan-agius4) [reply aio/content/guide/deployment.md]: Fantastic it works :)

**Comment by alan-agius4** [comment]
Caretaker: the above approval should be enough since the change here is trivial and in the majority of cases only effects formatting.

**Pull request closed without merging**